### PR TITLE
Multiple metrics

### PR DESCRIPTION
**Pull request by edancanning**
# Changes
* Please put your changes here

# Security Implications
* Please put any possible security-related concerns here. Feel free to @andykram if you're unsure. If none, please put "None"

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+        message: 'Please use "gql/queries" instead',
+      },
+    ],
+
+    // note you must disable the base rule as it can report incorrect errors
+    'no-unused-vars': 'off',
+    '@typescript-eslint/no-unused-vars': ['error'],
+
+    // Turning this off because next/link expects an invalid anchor
+    'jsx-a11y/anchor-is-valid': 'off',
```
Comment (erikvorhes) [inline .eslintrc.js:73]: Are these relevant to this repo?

**Review by erikvorhes — COMMENTED** [review]
EXCITING

File: .eslintrc.js (hunk 1 above)
Comment (edancanning) [reply .eslintrc.js:73]: No idea how that got in...

Comment (erikvorhes) [reply .eslintrc.js:73]: `copypasta`

**Review by erikvorhes — APPROVED** [review]
Approving, assuming those questionable lint rules are excised before merging. :)

File: .eslintrc.js (hunk 1 above)
Comment (edancanning) [reply .eslintrc.js:73]: Oh... it wasn't me... I won't name names though...

Comment (erikvorhes) [reply .eslintrc.js:73]: Wild, they don't seem to be in the file any more. WTH

**Pull request merged**